pkg/certificate/peer.go

```diff
@@ -31,8 +32,9 @@ type Config struct {
 	Identity identity.Config
 	Server   server.Config
 
-	Signer         identity.FullCAConfig
-	Authorizations authorization.Config
+	Signer            identity.FullCAConfig
+	AuthorizationDB   authorization.DBConfig
+	AuthorizationAddr string
```

Comment:
Shouldnt this be an exported config flag?

Reply:
It it, it just needs default and help tags.